python/docs/source/migration_guide/pyspark_upgrade.rst

```diff
@@ -42,6 +42,8 @@ Upgrading from PySpark 3.5 to 4.0
 * In Spark 4.0, ``squeeze`` parameter from ``ps.read_csv`` and ``ps.read_excel`` has been removed from pandas API on Spark.
 * In Spark 4.0, ``null_counts`` parameter from ``DataFrame.info`` has been removed from pandas API on Spark, use ``show_counts`` instead.
 * In Spark 4.0, the result of ``MultiIndex.append`` does not keep the index names from pandas API on Spark.
```

Comment:
Can we add a line her, where we tell users to have pandas version 2.1.0 installed for spark 4.0  
The only way now to find witch pandas version to install is to check the docker file in dev/infra 

https://github.com/jupyter/docker-stacks/blob/52a999a554fe42951e017f7be132d808695a1261/images/pyspark-notebook/Dockerfile#L69

Reply:
Good idea. Related information has been added to the top of the migration guide. Thanks!